build_runner/tool/create_test_all.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
```

Comment:
Turns out it's easier to duplicate it. It needs to be under one of the packages with a pubspec.yaml...